app/src/processing/app/syntax/JEditTextArea.java

```diff
@@ -163,12 +162,14 @@ public void mouseWheelMoved(MouseWheelEvent e) {
 
             // inertia scrolling on OS X will fire several shift-wheel events
             // that are negative values.. this makes the scrolling area jump.
-            boolean isHorizontal = Base.isMacOS() && e.isShiftDown();
```

Comment:
there must be some problem while merging...cuz i had to resolve conflicts manually...i  will correct it